src/client/wrappers/loadAndErrorWrapper/index.js

```diff
@@ -1,78 +1,37 @@
-import React, { Fragment, useContext } from 'react'
+import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 
-import { BoxContext } from '../box'
 import Initializing from './Initializing'
 import LoadingPublicProfile from './LoadingPublicProfile'
 import ErrorState from './Error'
 import UnlockingBox from './UnlockingBox'
+import NoProfile from './NoProfile'
+import { useProfileStates } from '../../hooks'
 
-const LoadAndErrorWrapper = props => {
-  const syncing = false
+const LoadAndErrorWrapper = ({ children, ethereumAddress }) => {
+  const {
+    loadingPublicProf,
+    unlockingProf,
+    noPublicProfileFound,
+    error,
+  } = useProfileStates()
 
-  const { boxes } = useContext(BoxContext)
+  // it was hardcoded as syncing=false before
+  const isInitializing = false
```

Comment:
Let's fix this in this PR - this should be taken from the `syncing` variable given from the`useAragonApi` hook

Reply:
I tried, including installing api packages, but useAragonApi expects to live in an application that communicates with Wrapper, not one that is inserted straight into it?